README.md

```diff
@@ -54,11 +54,13 @@ vars:
     zendesk_database: your_destination_name
     zendesk_schema: your_schema_name 
 ```
-### Step 4: Disable models for non-existent sources
-This package takes into consideration that not every Zendesk Support account utilizes the `schedule`, `domain_name`, `user_tag`, `organization_tag`, or `ticket_form_history` features, and allows you to disable the corresponding functionality. By default, all variables' values are assumed to be `true`. Add variables for only the tables you want to disable:
+### Step 4: Enable/Disable models for non-existent sources
+This package takes into consideration that not every Zendesk Support account utilizes the `schedule`, `schedule_holiday`, `audit_log`, `domain_name`, `user_tag`, `organization_tag`, or `ticket_form_history` features, and allows you to disable the corresponding functionality. By default, all variables' values are assumed to be `true`, except for `using_schedule_histories`. Add variables for only the tables you want to enable/disable:
```

Comment:
I noticed you removed the ticket_schedule, daylight_time, and time_zone. However, these are all dependent on the `using_schedules` variable ([reference](https://github.com/fivetran/dbt_zendesk_source/blob/fd7ffca469b7660e64e21ece0ddba7d757b18a24/models/stg_zendesk__time_zone.sql#L2)). Can we still include these sources and maybe it would help to call them out in the comment of the variable shown in the example below.

Reply:
Disregard this comment. I see you included them